MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Spacing](images/customization/maui-text-spacing.png)
+
+### Horizontal text alignment
+
+The horizontal alignment of the text can be controlled by `TextAlignment` property of barcode generator. The displayed value can be positioned at `Start`, `Center` or `End` of the control. The default value of `TextAlignment` property is center.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+       <barcode:SfBarcodeGenerator Value="12634" 
+                                   HeightRequest="150"
+                                   WidthRequest="240" 
+                                   ShowText="True" 
+                                   TextAlignment="End"/>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 240;
+        barcode.Value = "12634";
+        barcode.ShowText = true;
+        barcode.TextAlignment = TextAlignment.End;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Alignment](images/customization/maui-text-alignment.png)
+
+## Bar customization
+
+### Bar width customization
+
+Both the one dimensional and the two dimensional symbology support the `Module` property. The property is used to define the size of the smallest line or dot of the barcode.
+
+For one dimensional barcode, if this property is not set, the size of the smallest bar line is determined depending on the width available.
+
+The following code snippet shows the one dimensional barcode with `Module` property,
+
+{% tabs %}
+
+{% highlight xaml hl_lines="7" %}
+
+            <barcode:SfBarcodeGenerator Value="123456789" 
+                                        HeightRequest="150"
+                                        WidthRequest="240" 
+                                        ShowText="True" 
+                                        BackgroundColor="LightCyan">
+                <barcode:SfBarcodeGenerator.Symbology>
+                    <barcode:Codabar Module="1"/>
+                </barcode:SfBarcodeGenerator.Symbology>
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 240;
+        barcode.Value = "123456789";
+        barcode.ShowText = true;
+        barcode.BackgroundColor = Colors.LightCyan;
+        barcode.Symbology = new Codabar() { Module = 1 };
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Barcode Width Customization](images/customization/maui-barcode-width-customization.png)
+
+N> In the image above, the smallest bar line width is 1 logical pixel.
+
+Below code snippet shows the one dimensional barcode without the `Module` property,
```

Comment:
The following code sample shows the one-dimensional barcode without the `Module` property.

Reply:
Changes updated.